### ISSUE TITLE
Pretrained model doesn't give the predictions shown in the samples 

### ISSUE THREAD

**Issue by Hisan-007** [geyuying/PF-AFN#62]
I used the pre-trained model to get the predictions on the test set without any changes. Even though i used everything as it is, i'm getting very bad results with the pre-trained model. I used the images provided in the pf-afn-test/dataset folder. 

Here are the results:
![5](https://user-images.githubusercontent.com/76820944/156659634-8500d60b-5bdc-49cf-864a-49c48bb2ed59.jpg)
![4](https://user-images.githubusercontent.com/76820944/156659643-ab4c68c9-6979-4813-a891-ee9d5ae2cae2.jpg)
![3](https://user-images.githubusercontent.com/76820944/156659649-682479df-270d-40d0-85e6-79e2d79dfa3c.jpg)

I have no idea why this is happening, the results on their own provided test set should be similar to the samples images they have shown us.

**Comment by geyuying** [comment]
https://github.com/geyuying/PF-AFN/issues/3

**Comment by devkaranjoshi** [comment]
Can you give more details about your problem?

**Comment by albertigno** [comment]
I'm getting the exact same performance... I'm using a more up-to-date environment, but that shouldn't be a problem, isn't it? I'm running it on a RTX3060 with cuda 11.6, cupy 12.1.0 pytorch 1.12.0
